paymentsheet/src/main/res/values-fil/strings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Still need translation for this string in payments-core for the following new languages:
`Total: %s`

fil
hr
id
ms-rMY
no
nn-rNO
th

Reply:
Those are not supported yet. I'll remove from here.

Reply:
Actually they don't need to be removed from this module, there's no harm in keeping them.